adapters/adapters-commons/src/test/java/uk/gov/justice/raml/common/mapper/ActionMappingTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.raml.common.mapper;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.beans.HasPropertyWithValue.hasProperty;
+import static org.hamcrest.core.AllOf.allOf;
+import static org.hamcrest.core.IsCollectionContaining.hasItems;
+import static org.junit.Assert.assertThat;
+import static org.raml.model.ActionType.GET;
+import static org.raml.model.ActionType.POST;
+
+import java.util.List;
+
+import org.junit.Test;
+
+public class ActionMappingTest {
+
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
+    @Test
+    public void shouldCreateSingleMappingWithRequestTypeFromString() throws Exception {
+
+        List<ActionMapping> mappings = ActionMapping.listOf("..." + LINE_SEPARATOR +
```

Comment:
Can we move this into a private method to generate the Action Mapping String passing in the few different parameters that are different on each test?